packages/react-storage/src/components/StorageBrowser/views/index.ts

```diff
@@ -6,12 +6,18 @@ export {
   DeleteView,
   DeleteViewType,
   LocationActionView,
+  LocationActionViewProps,
```

Comment:
Do you want to vend the copy/delete/etc ViewProps too?

Reply:
We need to vend WAY more types than we currently are, just brought these up a level since i was in the file getting the compsable comp types